packages/react-components/package.json

```diff
@@ -35,8 +35,7 @@
     "@types/react": "16.9.42",
     "@types/react-dom": "16.9.10",
     "react": "16.8.6",
-    "react-dom": "16.8.6",
-    "@fluentui/babel-make-styles": "9.0.0-beta.4"
```

Comment:
These are no styles, this package is not needed

Reply:
but there are styles in all stories + accumulated stories from other packages. I'd say this should stay

Reply:
After your comment I restored configs (0d77b17), but it's still don't needed as Griffel package are declared on repo root.